### DISCUSSION TITLE
Abandoned?

### DISCUSSION THREAD

**Discussion by ikatun** [samdenty/gqless#246] [General]
Has this library been abandoned? The idea seems awesome, were there some conceptual issues in real-life usage with this approach that blocked the adoption? Even the docs for the lib look great

**Comment by vicary** [comment]
[GQty](https://github.com/gqty-dev/gqty) is the successor of gqless, and we are in active development.

Head over to [our website](https://gqty.dev) and try out the canary, we also have a Discord for questions and sharing!
